.github/scripts/mega-app-copy-files.sh

```diff
@@ -91,9 +91,17 @@ if [[ "$FRAMEWORK" == 'vue' ]]; then
     fi
 fi
 
-if [[ "$FRAMEWORK" == "react-native" && "$BUILD_TOOL" == 'cli' ]]; then
+if [[ "$FRAMEWORK" == "react-native" ]]; then
     echo "cp templates/components/react-native/cli/App.tsx mega-apps/${MEGA_APP_NAME}/App.tsx"
     cp templates/components/react-native/cli/App.tsx mega-apps/${MEGA_APP_NAME}/App.tsx
-    echo "cp templates/template-aws-exports.js mega-apps/${MEGA_APP_NAME}/aws-exports.js"
-    cp templates/template-aws-exports.js mega-apps/${MEGA_APP_NAME}/aws-exports.js
+    if [[ "$BUILD_TOOL" == 'cli' ]]; then
+        # React Native CLI has to use the template-aws-exports.js, which is an emplty aws-exports file
+        # Because we need to see the "'AuthError -" to know the errors are successfully logged.
+        # Because React Native CLI doesn't log error in the initial terminal. We have to use react-native-log-ios or log-android to log errors.
```

Comment:
Yes, that explains why we copy an empty _template-react-native-aws-exports.js_ instead of _template-aws-exports.js_....oh, I there's a typo on line 98, it should be _template-react-native-aws-exports.js_